ev-dev-tools/src/ev_cli/templates/interface-Exports.hpp.j2

```diff
@@ -3,13 +3,13 @@
 #ifndef {{ info.hpp_guard }}
 #define {{ info.hpp_guard }}
 
-{{ print_template_info('0.0.2') }}
+{{ print_template_info('1.0.0') }}
```

Comment:
Because I think we should use semantic versioning instead of arbitrary version numbers.
Semantic versioning is kind of a standard especially for open source projects.

Reply:
I also think semantic versioning should be something to aim for, but maybe its a bit premature to signal any sort of "stability" with the jump to 1.0.0 - especially if version 2.0.0 might be right around the corner,

Reply:
I agree with both of you, we should have started from -10.0.0